2022-11-29-mroonga.md

```diff
@@ -0,0 +1,94 @@
+    Now we can specify a tag for highlighting with open_tag and close_tag.
+    It was inconvinient for us to unable specifying different tags or chaning `class` because the fixed tag <span class="keyword">...</span> was used. 
+    Now, this new parameter provide more convenience for us because it allows specifying tags to highlight,such as <mark>...</mark>.
+    <!--今までは固定で <span class="keyword">...</span> というタグを使用しており、 class の変更や、別のタグが指定できず不便でした。 今回からタグを指定できるようになったので、 <mark>...</mark> などのハイライト用のタグも使えるようになって便利になりました。-->
+
+ .. code-block:: sql
+ 
+SELECT mroonga_highlight_html('Mroonga is the Groonga based storage engine.', 'groonga','<mark>' AS open_tag, '</mark>' AS close_tag) AS highlighted;
+
+-- +-----------------------------------------------------------+
+-- | highlighted                                               |
+-- +-----------------------------------------------------------+
+-- | Mroonga is the <mark>Groonga</mark> based storage engine. |
+-- +-----------------------------------------------------------+
+Please refer to :doc:`/reference/udf/mroonga_highlight_html` for details.
+
+* Added support for reference count mode.
+  Though this feature can keep fixed memory usage, its performance would be dropped. Thus, memory increments should be first consideration before use.
+    <!-- この機能によりメモリー使用量を一定量に保つことができますが、パフォーマンスが悪化します。 そのため、メモリー不足の場合には、この機能を使用する前に、メモリーを増強することを検討してください。-->
+
+  The reference count mode is used with `table_open_cache` <https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache> of MySQL.
+    <!-- 参照カウントモードは MySQLの table_open_cache <https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache> とともに使用します。（My SQLのというのがMy SQLのシステム上でという事ならonかinだと思います )-->
+  MySQL can cache specified number of tables  with `table_open_cache`. Groonga's object would not be released because the cached tables are still in use.
+  <!-- MySQLは table_open_cache で指定した個数のテーブルをキャッシュしておくことができます。キャッシュされているテーブルはまだ使用中なので、Groongaのオブジェクトも解放されません。-->
+  
+  The tables with low usage would be closed if number of open tables is larger than number specified with `table_open_cache`. Groonga objects would be released at the same time with table closing if the reference count mode is enabled.
+  
+<!--table_open_cache で指定した個数よりも多いテーブルが開かれたとき、使用頻度が低いテーブルが閉じられます。参照カウントモードが有効なとき、そのタイミングでGroongaのオブジェクトも閉じられます。
+【吉本】ここの日本語がよくわからなかったです。　Groongaのオブジェクトが開放されるのとオブジェクトが閉じるのは同義ですか？この説明は「どうやってreferencecount modeがメモリー使用量を一定に保つのか？」を説明しているという事であっていますか？-->
+
+
+ These are how the reference count mode keep fixed memory usage.
+
+For actual use, we need to adjust balance between memory usage and perfomance with value of `table_open_cache` while checking memory usage and value of static variables Open Tables<https://dev.mysql.com/doc/refman/8.0/en/server-status-variables.html#statvar_Open_tables>. Because this feature can keep fixed memory usage, but its performance would be dropped.
+
```

Comment:
https://github.com/yoshimotoyuk/yoshimotoyuk/pull/2/files#r1034336521
で教えていただいたことと英語の自然な流れで次へ（バランス調整する）をつなげるのに
下記文章たしてみました

For actual use, we need to adjust balance between memory usage and perfomance with value of `table_open_cache` while checking memory usage and value of static variables Open Tables<https://dev.mysql.com/doc/refman/8.0/en/server-status-variables.html#statvar_Open_tables>. Because this feature can keep fixed memory usage, but its performance would be dropped.

Reply:
良さそうに思いました！